PowerShellGet/PSModule.psm1

```diff
@@ -1881,11 +2002,13 @@ function Install-Module
 
                 $PSBoundParameters["Name"] = $psgetModuleInfo.Name
                 $PSBoundParameters["RequiredVersion"] = $psgetModuleInfo.Version
+                $PSBoundParameters[$script:AllowPrereleaseVersions] = $psgetModuleInfo.AdditionalMetadata.IsPrerelease -match "true"
+                $null = $PSBoundParameters.Remove("AllowPrerelease")

```

Comment:
>$null = $PSBoundParameters.Remove("AllowPrerelease") [](start = 16, length = 52)

Minor: Consider removing this as AllowPrerelease is not part of InputObject parameterset.